src/mon/MonCommands.h

```diff
@@ -527,9 +527,10 @@ COMMAND("osd crush add " \
 	"add or update crushmap position and weight for <name> with <weight> and location <args>", \
 	"osd", "rw", "cli,rest")
 COMMAND("osd crush set-device-class " \
-	"name=id,type=CephOsdName " \
-	"name=class,type=CephString ", \
-	"set the <class> of the device <name>", \
+        "name=class,type=CephString " \
+	"name=ids,type=CephString,n=N", \
+	"set the <class> of the osd(s) <id> [<id>...]," \
+        "or use <all|any|*> to set all.", \
```

Comment:
https://github.com/ceph/ceph/pull/16221/commits/e4fe4984017f106dbfb26d50980f1b2513fbd09a
https://github.com/ceph/ceph/pull/16283/commits/03fe1a6a1b1d39fdb1a830278938c87478252178

The above two commits need to be updated if you test them together with this change, otherwise this one is good to merge. (And we can fix and retest the others later)